src/librustc_mir_build/hair/pattern/_match.rs

```diff
@@ -4,8 +4,8 @@
 //! This file includes the logic for exhaustiveness and usefulness checking for
 //! pattern-matching. Specifically, given a list of patterns for a type, we can
 //! tell whether:
-//! (a) the patterns cover every possible constructor for the type [exhaustiveness]
-//! (b) each pattern is necessary [usefulness]
+//! (a) the patterns cover every possible constructor for the type exhaustiveness
+//! (b) each pattern is necessary usefulness
```

Comment:
Use parens here.

Reply:
What do those signify?

Reply:
I would say in this circumstance, the parenthesized word signifies how the word is being used in this context. That is, the phrase preceding the parenthetical is the definition for the word.

Reply:
I think I get it now... I thought those words were just continuations of the sentence